apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/schemas/Sgv2KeyspacesResourceImpl.java

```diff
@@ -42,6 +47,9 @@ public class Sgv2KeyspacesResourceImpl extends RestResourceBase
     implements Sgv2KeyspacesResourceApi {
   private static final Logger LOGGER = LoggerFactory.getLogger(Sgv2KeyspacesResourceImpl.class);
 
+  private static final Pattern BAD_DC_PATTERN =
```

Comment:
I think this is not correct.
If the DC name is passed without surrounding single quotes you will get this.
But if it is passed with single quotes then the message will be completely different:

```
cqlsh> create keyspace zupa WITH replication = {'class': ' = {'class': 'NetworkTopologyStrategy', dc1: 1};
NetworkTopologyStrategy  SimpleStrategy           
cqlsh> create keyspace zupa WITH replication = {'class': 'NetworkTopologyStrategy', 'xyz': 1};
ConfigurationException: Unknown data-center name 'xyz' passed to NetworkTopologyStrategy for keyspace 'zupa': it is either unknown to the configured snitch, or has no active member (known DCs: {Cassandra: 3 nodes})
cqlsh> create keyspace zupa WITH replication = {'class': 'NetworkTopologyStrategy', xyz: 1};
SyntaxException: line 1:80 no viable alternative at input ':' (... = {'class': 'NetworkTopologyStrategy', [xyz]:...)
```

Reply:
@tlasica Keep in mind that the exception I saw was from Cassandra-4.0, not DSE. And Cassandra-3.11 did not even response with a failure. I did not realize fail messages vary that much and Stagate tests default to C-4.

Reply:
So it seems different backends can have different error message.

Maybe instead of trying to find most robust pattern it would be better to rely on the fact that this is `ConfigurationException`? And in such case I would simply pass whatever is after it. This would work well for every configuration exception.

I understand that this may be too "technical" but it will also be way easier to debug in case of a customer reported problem.

Reply:
Due to use of Bridge gRPC which handles all queries (data, schema) same way, I don't think it is possible to rely on `ConfigurationException` -- this type will not be available on REST API side.
Bridge may get this (although due to intermediate Persistence API that Stargate implements, I am not sure it gets this either).

So at this point I think this may have to do. I'd be happy to find a better way tho.

I have filed issue against DSE backend (3.11 will not really be further developed so seems pointless), so if that gets tackled this can be revisited.